test/cljstyle/format/indent_test.clj

```diff
@@ -78,10 +92,23 @@
           indent/reindent-lines {:indents indents}
           "(condp = (:k x)\n a?\n a\nb?\n      b)"
           "(condp = (:k x)\n  a?\n    a\n  b?\n    b)"))
+    (is (rule-reformatted?
+          indent/reindent-lines {:indents indents}
+          "(condp = (:k x) a?\n a\nb?\n      b)"
+          (str "(condp = (:k x) a?\n" (space 18) "a\n" (space 16) "b?\n" (space 18) "b)")))
+    (is (rule-reformatted?
+          indent/reindent-lines {:indents indents}
+          "(condp = (:k x) a? a\nb? b)"
+          (str "(condp = (:k x) a? a\n" (space 16) "b? b)")))
     (is (rule-reformatted?
           indent/reindent-lines {:indents indents}
           "(cond->\n  a? (a 123)\n  b? (b true))"
           "(cond->\n  a? (a 123)\n  b? (b true))"))
+    (is (rule-reformatted?
+          indent/reindent-lines {:indents indents}
+          ;; Note that this is parsed "wrong" as the expr-part is missing from the cond->
+          "(cond-> a? (a 123)\n  b?\n(b true))"
+          (str "(cond-> a? (a 123)\n" (space 13) "b?\n" (space 11) "(b true))")))
```

Comment:
This is some changed behaviour that I don't think would be possible to keep backwards compatibility with, afaik there's no way for the `stair-indentation` to tell if the form that's on the same line is the "leading form" (`expr` in `cond->`) or if it's a condition that's on the same line...

Reply:
Hm, this could be solved by only using block-indent as the base when the `idx` parameter is `0` which probably is reasonable.

Things like
```clojure
(cond-> x a? (a 123) 
                 b? (b true))
``` 
is quite silly after all and probably not something that should be encouraged.